dopex-io/elvarg
# Implement Notifi React Card

## Scope

Implement Notifi React Card

## Screenshots

![Untitled](https://github.com/dopex-io/elvarg/assets/26240001/e04ad502-6741-48bd-8b81-7c4c1f033c29)

## How to Test
 - connect wallet and set up the network to Arbitrum
 - you can see the react card modal shows up by clicking the bell icon in AppBar
 - follow the steps in the card for testing
